openhtf/phasemanager.py

```diff
@@ -44,17 +44,9 @@
 
 FLAGS = gflags.FLAGS
 gflags.DEFINE_integer('phase_default_timeout_ms', 3 * 60 * 1000,
-                      'Test phase timeout in ms', lower_bound=0)
-
-_LOG = logging.getLogger('htf.phasemanager')
-
-# Only use 'is' checks, as that does pointer comparison for strings. That makes
-# this the same as an object(), but useful when printed.
-DIDNT_FINISH = 'DIDNT_FINISH'
-
-configuration.Declare(
-    'blacklist_phases', 'Phase names to skip', default_value=[])
+                      'blacklist_phases', 'Phase names to skip', default_value=[])
```

Comment:
The first merge was not right, I have fixed my merge, forget to push to my fork afterwards.